addons/l10n_ar_withholding/models/l10n_ar_partner_withholding.py

```diff
@@ -0,0 +1,46 @@
+from odoo import models, fields
+# from odoo.exceptions import ValidationError
+import logging
+# from dateutil.relativedelta import relativedelta
+_logger = logging.getLogger(__name__)
+
+
+class L10nArPartnerTax(models.Model):
+    _name = "l10n_ar.partner.tax"
+    _description = "Argentinean Partner Taxes"
+    _order = "to_date desc, from_date desc, tax_id"
+
+    partner_id = fields.Many2one(
+        'res.partner',
+        required=True,
+        ondelete='cascade',
+    )
+    tax_id = fields.Many2one(
+        'account.tax',
+        # domain=[('applicability', '=', 'taxes')],
+        required=True,
+        # change_default=True,
+    )
+    company_id = fields.Many2one(
+        related='tax_id.company_id', store=True,
+    )
+    from_date = fields.Date(
```

Comment:
Please fill in params

Reply:
@pablohmontenegro don't forget about this one.
Thanks!

Reply:
Thanks @vbe-odoo , what is recommended to fill in from_date, to_date and ref fields. Do I have to fill with a helper?

Reply:
@jco-odoo can you help me with this question from @pablohmontenegro ?
Thanks!

Reply:
string="From Date" at least